agent/grpc-middleware/rate_limit_mappings.gen.go

```diff
@@ -0,0 +1,27 @@
+// generated by protoc-gen-consul-rate-limit; DO NOT EDIT.
+package middleware
+
+import "github.com/hashicorp/consul/agent/consul/rate"
+
+var rpcRateLimitSpecs = map[string]rate.OperationType{
+	"/hashicorp.consul.acl.ACLService/Login":                                     rate.OperationTypeWrite,
+	"/hashicorp.consul.acl.ACLService/Logout":                                    rate.OperationTypeWrite,
+	"/hashicorp.consul.connectca.ConnectCAService/Sign":                          rate.OperationTypeWrite,
+	"/hashicorp.consul.connectca.ConnectCAService/WatchRoots":                    rate.OperationTypeRead,
+	"/hashicorp.consul.dataplane.DataplaneService/GetEnvoyBootstrapParams":       rate.OperationTypeRead,
+	"/hashicorp.consul.dataplane.DataplaneService/GetSupportedDataplaneFeatures": rate.OperationTypeRead,
+	"/hashicorp.consul.dns.DNSService/Query":                                     rate.OperationTypeRead,
+	"/hashicorp.consul.internal.operator.OperatorService/TransferLeader":         rate.OperationTypeWrite,
```

Comment:
I think this should not be rate limited. We have 2 choices here:

- add another Type (eg: bypass) 
- remove it from the list and assume if not in the list it means bypass

I vote for option 1 if it's not too much work

Reply:
Yep makes sense. It looks like we added `OperationTypeExempt` in the net/rpc mapping, so I'll adopt that here.